src/components/Class/ClassCard/index.js

```diff
@@ -69,12 +69,18 @@ function ClassCard(props) {
             {moment(classEndTime).format("hh:mm a")}
           </p>
           {item.facilitator_id == user.data.user.id || flag ? (
-            <button
-              className="delete-button"
-              onClick={() => deleteHandler(item.id)}
-            >
-              Delete
-            </button>
+            <div className="class-card-actions">
+              <i
+                className="class-card-action-icon fa fa-trash"
+                onClick={() => deleteHandler(item.id)}
```

Comment:
@tarun-dugar  Can we please use handleClickOpen instead of deleteHandler .  because whenever I am going to delete the class. without clicking the class is deleting.

Reply:
nice catch! fixed it 👍